packages/atomic/src/components/search/facets/atomic-popover/atomic-popover.tsx

```diff
@@ -117,6 +132,19 @@ export class AtomicPopover implements InitializableComponent {
         >
           {this.facetLabel}
         </span>
+        <span
+          title={this.numberOfSelectedValues}
+          part="value-count"
+          class={`value-box-count truncate pl-0.5 w-auto mt-0 text-sm ${
+            this.isMenuVisible
+              ? 'text-primary'
+              : 'text-neutral-dark group-hover:text-primary-light group-focus:text-primary'
+          }`}
+        >
+          {this.bindings.i18n.t('between-parentheses', {
+            text: this.numberOfSelectedValues,
```

Comment:
We can use `toLocaleString` on the count number value like we do for other numbers that we display inside atomic

Reply:
sure